disable_deactivated_accounts.py

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/python
+import sys
+import optparse
+import requests
+import ldap
+import json
+
+from auth0_api_settings import AUTH0_API_KEY, AUTH0_API_URL, AUTH0_CONNECTION, LDAP_HOST,\
```

Comment:
This is done, with a config file that could potentially be shared with other scripts, by breaking out auth0 api config, ldap config, proxy config and specific script config in the json file. Let me know if that's acceptable.